eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -189,36 +212,40 @@ instance SafeToHash CostModel where
 
 instance HashWithCrypto CostModel CostModel
 
-instance NoThunks CostModel
+deriving via InspectHeapNamed "PV1.EvaluationContext" PV1.EvaluationContext instance NoThunks PV1.EvaluationContext
+
+deriving via InspectHeapNamed "CostModel" CostModel instance NoThunks CostModel
 
-instance NFData CostModel
+instance NFData PV1.EvaluationContext where
+  rnf = undefined
```

Comment:
I think we will need Plutus to provide this instance.

Reply:
I'll update to the tip of `release/1.0.0` (in Plutus) once https://github.com/input-output-hk/plutus/pull/4471 lands.